### ISSUE TITLE
Read support for population calls

### ISSUE THREAD

**Issue by lorihandley** [fritzsedlazeck/Sniffles#102]
Hi Fritz,
I am confused by the read support data when I try to call the variants for a trio.  I have followed the method you outlined (single Sniffles runs, SURVIVOR merge, second Sniffles runs using the Ivcf and then second SURVIVOR merge to get a multisample vcf); however, the number of reads supporting the calls in the final vcf is confusing me.  For instance, I have this call in one of my vcfs:

`22 46883505     22564   N       <INS>   .     PASS      IMPRECISE;SVMETHOD=Snifflesv1.0.9;CHR2=22;END=46883775;ZMW=20;STD_quant_start=21.695622;STD_quant_stop=41.218928;Kurtosis_quant_start=1.984273;Kurtosis_quant_stop=2.108020;SVTYPE=INS;RNAMES=m54037_180402_215537/55313022/4338_31336,m54037_180402_215537/73401014/0_19376,m54037_180402_215537/8389356/0_24178,m54037_180403_080534/21692842/0_17376,m54037_180405_023104/54657469/0_13502,m54037_180405_124620/34800185/7342_26209,m54037_180405_124620/66650886/0_13285,m54037_180406_090927/10551568/0_32187,m54037_180501_135927/7274678/0_28927,m54037_180502_001718/49349293/11714_25137;SUPTYPE=AL,SR;SVLEN=269;STRANDS=+-;RE=20;AF=0.869565    GT:DR:DV        1/1:3:20`

where it appears that I have 20 reads supporting the alt call.  Neither of the other 2 samples in my trio have this variant call in their Sniffles output.  When I merge, recall and merge again, I end up with this entry:

`22 46883505     INS0032042SUR   N       <INS> . PASS    SUPP=1;SUPP_VEC=001;AVGLEN=295;SVTYPE=INS;SVMETHOD=SURVIVORv2;CHR2=22;END=46883775;CIPOS=0,0;CIEND=0,0;STRANDS=+-       GT:PSV:LN:DR:ST:TY:CO   0/0:NA
:295:0,0:+-:INS:22_46883505-22_46883775 0/0:NA:295:0,0:+-:INS:22_46883505-22_46883775   1/1:NA:295:0,2:+-:INS:22_46883505-22_46883775`

Why is there now a 0,2 for read support?  Should it not still be 3,20?

Thank you so much for your assistance.

**Comment by fritzsedlazeck** [comment]
Hi, 
yeah probably. I will need to check this. Survivor and Sniffles had been developed independently.
As you may know almost every SV caller reports these values differently (different tags) so its always hard to parse them. However, no excuse it should have been tested by me since I implemented both tools. 
Thanks for pointing in out. I hope I have a fix soon.
Fritz

**Comment by lorihandley** [comment]
Thanks for looking into it so quickly.  I definitely understand that the translations between different vcf tags is tricky!  One of the many reasons I am thankful for tools such as yours.

**Comment by grabowsp** [comment]
Hi Fritz,
I ran into the same problem. Any updates on a fix?
Thanks!
- Paul

**Comment by fritzsedlazeck** [comment]
Sorry that slipped off my table... 
can someone share the files or at least the first few entries? 
email: fritz.sedlazeck@gmail.com

**Comment by grabowsp** [comment]
Just emailed you files containing the top lines of the files.

Hope they are helpful. And thanks again for your help.

Paul

**Comment by fritzsedlazeck** [comment]
Thanks 
Fritz

**Comment by fritzsedlazeck** [comment]
I am really sorry that this slipped my mind @lorihandley and thanks for sending some files @grabowsp. I have updated the code now and it seems to work.
Please clone the current code, try it and let me know if that solves the problem.
Thank you
Fritz

**Comment by lorihandley** [comment]
Thanks for looking at this again, Fritz.  I did try running my samples again with the latest release of SURVIVOR, but I think that the issue may be coming in when I run the 2nd round of Sniffles using the merged vcf as the input.  

1st vcfs for single samples from Sniffles show:
no variant call at this location for sample 1
no variant call at this location for sample 2
22      46883505        22564   N       <INS>   .       PASS    IMPRECISE;SVMETHOD=Snifflesv1.0.9;CHR2=22;END=46883775;ZMW=20;STD_quant_start=21.695622;STD_quant_stop=41.218928;Kurtosis_quant_start=1.984273;Kurtosis_quant_stop=2.108020;SVTYPE=INS;SUPTYPE=AL,SR;SVLEN=269;STRANDS=+-;RE=20;AF=0.869565    GT:DR:DV    1/1:3:20

1st SURVIVOR merged vcf (I used 50bp as the max distance between breakpoints):
22      46883505        22564   N       <INS>   .       PASS    SUPP=2;SUPP_VEC=101;SVLEN=296;SVTYPE=INS;SVMETHOD=SURVIVORv2;CHR2=22;END=46883775;CIPOS=0,10;CIEND=0,21;STRANDS=+-      GT:PSV:LN:DR:ST:QV:TY:ID:RAL:AAL:CO     1/1:NA:322:1,5:+-:.:INS:25606:NA:NA:22_46883515-22_46883796  ./.:NaN:0:0,0:--:NaN:NaN:NaN:NAN:NAN:NAN        1/1:NA:269:3,20:+-:.:INS:22564:NA:NA:22_46883505-22_46883775

However, after the 2nd round of Sniffles is run for the single samples, the supporting reads change and are carried through SURVIVOR to the 2nd merged vcf, too:
22      46883505        31845   N       <INS>   .       PASS    IMPRECISE;SVMETHOD=Snifflesv1.0.9;CHR2=22;END=46883775;SVTYPE=INS;SUPTYPE=input;SVLEN=296;STRANDS=+-;SEQ=;RE=0;AF=-nan  GT:DR:DV        0/0:0:0
22      46883505        31845   N       <INS>   .       PASS    IMPRECISE;SVMETHOD=Snifflesv1.0.9;CHR2=22;END=46883775;SVTYPE=INS;SUPTYPE=input;SVLEN=296;STRANDS=+-;SEQ=;RE=0;AF=-nan  GT:DR:DV        0/0:0:0
22      46883505        31845   N       <INS>   .       PASS    IMPRECISE;SVMETHOD=Snifflesv1.0.9;CHR2=22;END=46883775;SVTYPE=INS;SUPTYPE=input;SVLEN=296;STRANDS=+-;SEQ=;RE=2;AF=1     GT:DR:DV        1/1:0:2

I am running Sniffles in the simple mode of 
sniffles -m <sorted.bam> -v <output.vcf> --Ivcf <1stMerged.vcf>
